js/radio.js

```diff
@@ -59,13 +59,20 @@
 		constructor: Radio,
 
 		destroy: function () {
-			this.$parent.remove();
 			// remove any external bindings
 			// [none]
 			// empty elements to return to original markup
 			// [none]
 			// return string of markup
-			return this.$parent[0].outerHTML;
+			if (this.$parent) {
```

Comment:
Well, actually we already do the check earlier in the file:

		if (this.$parent.length === 0) {
			this.$parent = null;
		}